userspace/libsinsp/parsers.cpp

```diff
@@ -1225,7 +1225,7 @@ void sinsp_parser::parse_clone_exit(sinsp_evt *evt)
 
 		tinfo->m_tty = ptinfo->m_tty;
 
-		tinfo->m_loginuid = ptinfo->m_loginuid;
```

Comment:
Not sure we can just drop loginuid

Reply:
This is superseeded by `tinfo->m_loginuser = ptinfo->m_loginuser;`